nibabel/freesurfer/io.py

```diff
@@ -160,6 +161,49 @@ def read_morph_data(filepath):
     return curv
 
 
+def write_morph_data(file_like, values, fnum=0):
+    """Write Freesurfer morphometry data `values` to file-like `file_like`
+
+    Equivalent to FreeSurfer's `write_curv.m`_
+
+    See also:
+    http://www.grahamwideman.com/gw/brain/fs/surfacefileformats.htm#CurvNew
+
+    .. _write_curv.m: \
+    https://github.com/neurodebian/freesurfer/blob/debian-sloppy/matlab/write_curv.m
+
+    Parameters
+    ----------
+    file_like : file-like
+        String containing path of file to be written, or file-like object, open
+        in binary write (`'wb'` mode, implementing the `write` method)
+    values : array-like
+        Surface morphometry values
+    fnum : int, optional
+        Number of faces in the associated surface
+    """
+    magic_bytes = np.array([255, 255, 255], dtype=np.uint8)
+
+    array = np.asarray(values).astype('>f4').squeeze()
```

Comment:
I suppose. Maybe it makes more sense to let people do their own squeezing, if that's what they want.

I would say allow (N, 1, 1) as well, though, since that's the default shape coming in from MGH or surface Nifti files.